core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerBuilder.java

```diff
@@ -71,8 +71,9 @@
     /**
      * Creates a new {@link CircuitBreakerBuilder} with the specified name.
      *
-     * @param name The name of the circuit breaker.
```

Comment:
I think there's no reason to remove this.

Reply:
Ok, I will revert all deletion of @param statements.